packages/itwinui-css/src/tabs/classes.scss

```diff
@@ -15,6 +14,18 @@
   }
 }
 
+.iui-tabs-actions-wrapper {
+  @include iui-tabs-actions-wrapper;
+
+  .iui-tabs-actions {
```

Comment:
does this need to be nested?

Reply:
Unnested [`977dbaf` (#860)](https://github.com/iTwin/iTwinUI/pull/860/commits/977dbaf46789dffaf67da1306912536e2c17376e)